animations/index.md

```diff
@@ -0,0 +1,158 @@
+to the message list.
+
+## Animation types
+
+Animations fall into one of two categories: tween- or physics-based.
+The following sections explain what these terms mean, and points you to
+resources where you can learn more. Note that, in some cases,
+the best documentation we currently have is example code in the
+Flutter gallery. More docs and examples will be forthcoming.
+
+### Tween animation
+
+Short for _in-betweening_. In a tween animation, the beginning
+and ending points are defined, as well as a timeline, and a curve
+that defines the timing and speed of the transition.
+The framework calculates how to transition from the beginning point
+to the end point.
+
+The docs listed above, such as the [animations
+tutorial](/tutorials/animation/) aren't about tweening,
+specifically, but they use tweens in their examples.
+
+### Physics-based animation
+
+In physics-based animation, motion is modeled to resemble real-world
+behavior. When you toss a ball, for example, where and when
+it lands depends on how fast it was tossed, how heavy it is, and how
+far off the ground. Similarly, dropping a ball attached to a spring falls
+(and bounces) differently than dropping a ball attached to a string.
+
+Note that more docs and examples on physics-based animations will be
+forthcoming.
+
+* [Flutter Gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery)<br>
+Under **Material Components**, the
+[Grid](https://github.com/flutter/flutter/blob/master/examples/flutter_gallery/lib/demo/material/grid_list_demo.dart) example
+demonstrates a fling animation. Select one of the images from
+the grid and zoom in. You can pan the image with flinging or dragging
+gestures.
+
+* Also see the API docs for
+[AnimationController.animateWith](https://docs.flutter.io/flutter/animation/AnimationController/animateWith.html) and
+[SpringSimulation](https://docs.flutter.io/flutter/physics/SpringSimulation-class.html).
+
+## Common animation patterns
+
+Most UX or motion designers find that certain animation patterns are
+used repeatedly when designing a UI. This section lists some of the commonly
+used animation patterns, and tells you where you can learn more.
+
+### Animated list or grid
+This pattern involves animating the addition or removal of elements from a
+list or grid.
+
+* [AnimatedList example](/catalog/samples/animated-list/)<br>
+This demo, from the [Sample App Catalog](/catalog/samples), shows how to
+animate adding an element to a list, or removing a selected element.
+The internal Dart list is synced as the user modifies the list using
+the plus (+) and minus (-) buttons.
+
+### Shared element transition
+
+In this pattern, the user selects an element&mdash;often an
+image&mdash;from the page, and the UI animates the selected element
+to a new page with more detail. In Flutter, you can easily implement
+shared element transitions between routes (pages) using the Hero widget.
```

Comment:
would it make sense, here and elsewhere, to link the class names to the API docs for those classes? Or is it better to only have the "also see" links below?